integrations/terraform/modules/data_products/modules.tf

```diff
@@ -15,14 +13,25 @@ module "data_access" {
   datasets            = var.datasets
 }
 
+module "service_access" {
```

Comment:
what is your reasoning behind migrating the Athena access to a new module? I am not against it per se but wanted to hear your view

Reply:
Before the refactor, this entire part about service access was included in the 'roles' module and passed as an output to the 'users' module. To me it felt more natural and easier to reason about by encapsulating it in a seperate module, especially because it is used both in the 'roles' and the 'users' modules. It also brings it in line with the 'data_access' module.